ksql-engine/src/main/java/io/confluent/ksql/topic/TopicDeleteInjector.java

```diff
@@ -99,22 +101,21 @@ public TopicDeleteInjector(
             () -> topicClient.deleteTopics(ImmutableList.of(source.getKafkaTopicName())),
             ExecutorUtil.RetryBehaviour.ALWAYS);
       } catch (Exception e) {
-        throw new KsqlException("Could not delete the corresponding kafka topic: "
-            + source.getKafkaTopicName(), e);
+        throw new RuntimeException("Could not delete the corresponding kafka topic: "
```

Comment:
we should stick with `KsqlException` because we handle those properly at the top level - why did we change this? Same comment goes for the below changes

Reply:
@rodesai mentioned to use RuntimeExceptions instead, and also how KsqlException should only be used for errors indicating a problem with the user's statement.